### ISSUE TITLE
I think there are something unreasonably

### ISSUE THREAD

**Issue by qwb492859377** [muesli/cache2go#13]
this is my fork diff :https://github.com/qwb492859377/cache2go/commit/87ff22ffdbcaf9e4f808793847db5557b85ec596

I think in addInternal() function, it should compare the timer start time, not duration time.

**Comment by muesli** [comment]
I agree, a timer would be a lot nicer here. Would you mind opening a proper pull request for your proposed changes?

**Comment by qwb2333** [comment]
I use my new account which pull requests.And after I will use this account along.

**Comment by vaibhavsanap** [comment]
HI @muesli ,
         Can I work on this.

**Comment by muesli** [comment]
@vaibhavsanap Of course!
